### PR TITLE
add phase 1 unbond button and hook up modal

### PR DESCRIPTION
**Pull request by jeremy-babylonlabs**
Screenshot 2024-12-13 at 8 07 47 PM

did some refactoring along the way to clean it up
button looks abit different from original design, will need to fix the core-ui for this. but this shouldn't block the merge

### CODE REVIEW

File: src/app/components/Delegations/Delegation.tsx [hunk 1]
```diff
@@ -77,110 +81,48 @@ export const Delegation: React.FC<DelegationProps> = ({
...
-  const renderState = () => {
+  const renderState = () =>
     // overflow should be shown only on active state
-    if (isOverflow && isActive) {
-      return getState(DelegationState.OVERFLOW);
-    } else {
-      return getState(intermediateState || state);
-    }
-  };
+    isOverflow && isActive
```
Comment (gbarkhatov) [inline src/app/components/Delegations/Delegation.tsx:90]: Can we simplify this a little bit

File: src/app/components/Delegations/Delegation.tsx [hunk 2]
```diff
@@ -42,18 +46,18 @@ export const Delegation: React.FC<DelegationProps> = ({
...
-  const { getFinalityProviderMoniker } = useFinalityProviderService(); // get the moniker of the finality provider
-  useEffect(() => {
-    const timerId = setInterval(() => {
-      setCurrentTime(Date.now());
-    }, 60000); // set the refresh interval to 60 seconds
+  const { getFinalityProviderMoniker } = useFinalityProviderService();
+  const { coinName, mempoolApiUrl } = getNetworkConfig();
 
+  useEffect(() => {
+    const timerId = setInterval(() => setCurrentTime(Date.now()), 60000); // Update every minute
```
Comment (gbarkhatov) [inline src/app/components/Delegations/Delegation.tsx:60]: Let's use `ONE_MINUTE` constant that we have in `src/app/constants`

**Review by gbarkhatov — APPROVED** [review]
Besides @jrwbabylonlab comments and the implementation looks correct in terms of UI

**Comment by totraev** [comment]
> will need to fix the core-ui for this

@jeremy-babylonlabs lets discuss this before you start

**Pull request merged**